backend/src/packages/sites/libs/enums/section-type-to-prompt.enum.ts

```diff
@@ -0,0 +1,26 @@
+import { type ValueOf } from '~/libs/types/types.js';
+import { SectionType } from '~/packages/sections/libs/enums/enums.js';
+
+const SectionTypeToPrompt: Record<
```

Comment:
```suggestion
const sectionTypeToPrompt: Record<
```
seems like map

Reply:
https://binary-studio.com/2021/08/04/js-and-everyday-data-structures/

Reply:
got it, nice post btw